tests/config/ray-cluster.mini.yaml.template

```diff
@@ -56,58 +36,18 @@ spec:
       minReplicas: 1
       maxReplicas: 1
       groupName: small-group
-      # the following params are used to complete the ray start: ray start --block --node-ip-address= ...
+      # the following params are used to complete the ray start: ray start --num-cpus=1 --block
       rayStartParams:
-        redis-password: 'LetMeInRay' # Deprecated since Ray 1.11 due to GCS bootstrapping enabled
-        node-ip-address: $$MY_POD_IP
         block: 'true'
+        num-cpus: '1'
       #pod template
       template:
-        metadata:
-          labels:
-            rayCluster: raycluster-compatibility-test
```

Comment:
`rayCluster: raycluster-compatibility-test` cannot be removed in this PR. It is used in the NodePort service.

https://github.com/ray-project/kuberay/blob/3a647129e248640f2314bfafd257f6b7a5db232f/tests/config/raycluster-service.yaml#L8